src/Akeneo/Platform/Bundle/InstallerBundle/Command/DatabaseCommand.php

```diff
@@ -58,11 +55,11 @@ protected function configure()
     {
         $this
             ->addOption(
-                'fixtures',
+                'skip-fixtures',
```

Comment:
```suggestion
                'fixtures-to-skip',
```
Maybe more understandable?